changelog/5957.deprecation.rst

```diff
@@ -0,0 +1,3 @@
+Deprecate `~sunpy.image.coalignment` as the code has now been moved to
```

Comment:
You should mention here that all functionality has been replicated with the same API in `sunkit_image.coalignment`. (if it has, I hope it has...) Also mention the motivation for doing this? Users are likely to grumble less if we explain to them why we're making their lives harder.

Also remove the tidles, or this will show up as "Deprecated coalignment, code has been moved to coalignment"!!

Reply:
Yes, the API is identical. I've added a "What's New" entry that gives some explanation as to the motivation as I figured this would be more visible to users than a changelog.